src/themes/dojo/select.m.css

```diff
@@ -13,53 +15,41 @@
 /* custom input styles */
 .trigger {
 	background-color: transparent;
-	border: 3px solid var(--input-border-color);
+	border: var(--border-width) solid var(--color-border);
+	border-bottom-color: var(--color-border-strong);
 	font-size: inherit;
+	line-height: var(--line-height-base);
 	min-width: 200px;
```

Comment:
Why the min width? Also, are we making all inputs take up 100% of their parent, but giving them a uniform `max-width`?